docs/en/7D2D/modding.md

```diff
@@ -0,0 +1,82 @@
+---
+title: 'Important Modding Info'
```

Comment:
Can we change this to "Important modding info"? 
To be in line with the other sidenav titles

Reply:
Yep glad you spotted that - standards are important!